### ISSUE TITLE
Type hints cause crash

### ISSUE THREAD

**Issue by kc9jud** [Technologicat/pyan#42]
I think something from the `typing` module needs to be imported... I get this error when trying to run `pyan3`:

```
pyan3 
Traceback (most recent call last):
  File "/home/pjfasano/.local/bin/pyan3", line 5, in <module>
    from pyan.main import main
  File "/home/pjfasano/.local/lib/python3.6/site-packages/pyan/__init__.py", line 26, in <module>
    grouped: bool = True,
NameError: name 'Union' is not defined
```

**Comment by Technologicat** [comment]
Fixed by #43.

**Comment by Technologicat** [comment]
PyPI package `pyan3` updated. Please try again with the latest version, and reopen this issue if necessary.
